roxctl/image/sbom/sbom.go

```diff
@@ -0,0 +1,123 @@
+package sbom
+
+import (
+	"bytes"
+	"encoding/json"
+	"io"
+	"net/http"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	imageUtils "github.com/stackrox/rox/pkg/images/utils"
+	"github.com/stackrox/rox/pkg/utils"
+	"github.com/stackrox/rox/roxctl/common"
+	"github.com/stackrox/rox/roxctl/common/environment"
+	"github.com/stackrox/rox/roxctl/common/flags"
+	"github.com/stackrox/rox/roxctl/common/util"
+)
+
+const (
+	imageSBOMAPIPath = "/api/v1/images/sbom"
+)
+
+// Command sends a request to Central to generate an SBOM from an image scan.
+func Command(cliEnvironment environment.Environment) *cobra.Command {
+	imageSBOMCmd := &imageSBOMCommand{env: cliEnvironment}
+
+	c := &cobra.Command{
+		Use:   "sbom",
+		Short: "Generate an SBOM from an image scan.",
+		Long:  "Generate an SBOM from an image scan. Optionally, force a rescan of the image. You must have write permissions for the `Image` resource.",
+		RunE: util.RunENoArgs(func(c *cobra.Command) error {
+			if err := imageSBOMCmd.construct(c); err != nil {
+				return err
+			}
+
+			return imageSBOMCmd.GenerateSBOM()
+		}),
+	}
+
+	c.Flags().StringVarP(&imageSBOMCmd.image, "image", "i", "", "Image name and reference. (e.g. nginx:latest or nginx@sha256:...)")
+	c.Flags().BoolVarP(&imageSBOMCmd.force, "force", "f", false, "The --force flag ignores Central's cache for the scan and forces a fresh re-pull from Scanner")
```

Comment:
```suggestion
	c.Flags().BoolVarP(&imageSBOMCmd.force, "force", "f", false, "Ignores Central's cache for the scan and forces a fresh re-pull from Scanner")
```
Imo repeating the flag in the description is redundant.